CHANGELOG/CHANGELOG-1.6.md

```diff
@@ -0,0 +1,110 @@
+# Changelog 1.6
+
+## 1.6.0 - 2022-03-29
+
+The Egress feature is graduated from Alpha to Beta and is therefore enabled by default.
+
+### Added
+
+- Add support for the following capabilities for [Antrea IPAM] feature:
+  * Support pre-allocating continuous IPs for StatefulSet. ([#3281](https://github.com/antrea-io/antrea/pull/3281), [@annakhm])
+  * Support specifying VLAN for IPPool. Traffic from the Pods whose IPPool is configured with VLAN ID will be tagged when leaving Node uplink. ([#3247](https://github.com/antrea-io/antrea/pull/3247), [@gran-vmv])
+- Add support for the following capabilities for [Antrea Multi-cluster] feature:
+  * Add ACNP copy span for multi-cluster. ([#3363](https://github.com/antrea-io/antrea/pull/3363), [@Dyanngg])
```

Comment:
Do not call it "copy span" as it is not a well-known term. Say: "Support Antrea ClusterNetworkPolicy replication in a ClusterSet"